.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -62,13 +55,15 @@ Release of binaries on conda-forge
 To ensure CADET-Core is accessible to a broad community, it is available as a Python package on conda-forge.
 Other software, such as our frontend, `CADET-Process`, and our Python interface, `CADET-Python`, import this package.
 
-- [ ] Go to github.com/conda-forge/cadet-feedstock
-- [ ] Create a new branch and open a PR:
+- [ ] Go to your fork of github.com/conda-forge/cadet-feedstock or create the fork
+- [ ] Create a new branch on your fork and open a PR:
 - [ ] Change the recipe/meta.yaml file:
```

Comment:
Note, most of these are also part of the checklist that is auto-generated when opening the PR.